llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -622,6 +620,12 @@ def IREELinalgExt_AttentionOp : IREELinalgExt_Op<"attention",
     int64_t getOutputRank() {
       return getOutputType().getRank();
     }
+    int64_t getMaxRank() {
```

Comment:
Either assert that number of results == 3 or return `std::optional<int64_t>` (similar to `getMaxType` above)